_aggregations/pipeline/stats-bucket.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: default
+title: Stats bucket
+parent: Pipeline aggregations
+nav_order: 180
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#stats_bucket-extended_stats_bucket/
+  - /query-dsl/aggregations/pipeline/stats-bucket/
+---
+
+# Stats bucket aggregations
+
+The `stats_bucket` aggregation is a sibling aggregation that returns a variety of stats (`count`, `min`, `max`, `avg`, and `sum`) for the buckets of a previous aggregation.
```

Comment:
is "sibling aggregation" a know type of aggregation? how is it different from calling it pipeline aggregation?

Reply:
@rishabhmaurya 
Sibling and parent aggregations are defined on the main pipeline aggregations page. They're defined by which aggregations the pipeline aggregation takes as input and where it creates its result buckets.